app/templates/base.html

```diff
@@ -95,6 +100,48 @@
             }
         )
     })
+
+    const user = "{{user.username}}"
```

Comment:
user object is only available in user.html, it is not available in base.html. It will work for /user/<username> endpoint, but it will not work for other endpoints, you may get `jinja2.exceptions.UndefinedError`, please check.

Reply:
I suggest moving follow, unfollow links to some reusable html template and use it everywhere, and move this javascript code to the new template created.

Reply:
Hey @SriNandan33 I made those follow/unfollow buttons snippets, and migrated the inline javascript used for following/unfollowing into a static directory